tests/test_modules/psql_test.py

```diff
@@ -1,4 +1,21 @@
 from subprocess import call, check_output
+import psycopg2
 
 class TestPsql:
-    """ Test that PostgreSQL is properly installed, up and running """
\ No newline at end of file
+    """ Test that PostgreSQL is properly installed, up and running """
+    def checkPresence(self):
+        output = check_output(["psql","--version"]).decode("utf-8").lstrip().rstrip()
+        print(output)
+        return output
+
+    def checkDatabase(self):
+        output = check_output(["psql", "-dfcc_provision", "-c select datname from pg_database"]).decode("utf-8").lstrip().rstrip()
```

Comment:
I gave a space, but it gave me the error : `psql: FATAL:  database " fcc_provision" does not exist`... maybe internally the library is splitting directly after `-d` ?